diff --git a/src/executors/test/lib/utils.js b/src/executors/test/lib/utils.js
index 303820fe488d425415078474f7b541488810ab16..ecd9ebfb953f0f29ad7300ae4cfd3351dda9b640 100644
--- a/src/executors/test/lib/utils.js
+++ b/src/executors/test/lib/utils.js
@@ -59,7 +59,7 @@ async function getExtraArgs(options) {
     const schema = await Promise.resolve().then(()=>require("../schema.json"));
     const extraArgs = {};
     for (const key of Object.keys(options)){
-        if (!schema.properties[key]) {
+        if (!schema.properties[key] || key === 'watch') {
             extraArgs[key] = options[key];
         }
     }
